### ISSUE TITLE
Upgrade CAP to Java 11 and double-check Maintenance

### ISSUE THREAD

**Issue by wwelling** [TAMULib/CAP#153]
UI
1. upgrade dependencies
2. update auth service
3. update/verify build and ci

Service
1. upgrade weaver
2. update auth service
3. update/verify build and ci

Update POM repositories:

```
    <repository>
      <id>tamu-releases</id>
      <url>https://artifacts.library.tamu.edu/repository/maven-releases/</url>
    </repository>

    <repository>
      <id>tamu-snapshots</id>
      <url>https://artifacts.library.tamu.edu/repository/maven-snapshots/</url>
    </repository>
```

Update Weaver Webservice Parent:

```
  <parent>
    <groupId>edu.tamu.weaver</groupId>
    <artifactId>webservice-parent</artifactId>
    <version>2.0.2</version>
  </parent>
```

Update Weaver UI Core:

`"@wvr/core": "2.0.6"`

**Comment by jcreel** [comment]
This card has lingered, and another upgrade is indicated.
Blocked by Weaver-Webservice-Core#119 and Weaver-UI-Core#160

**Comment by jcreel** [comment]
duplicate of https://github.com/TAMULib/CAP/issues/172
